server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java

```diff
@@ -243,6 +243,10 @@ public boolean isNoOpRecordSupported() {
         return this.isAtLeast(IBP_3_3_IV1);
     }
 
+    public boolean isApiForwardingSupported() {
```

Comment:
nit: Should we say "isApiForwardingEnabled" since we require API forwarding at this version?